e2e_test/source/cdc_inline/alter/cdc_table_alter.slt

```diff
@@ -0,0 +1,223 @@
+control substitution on
+
+system ok
+mysql -u root --protocol='tcp' -e "
+    SET GLOBAL time_zone = '+00:00';
+"
+
+system ok
+mysql -u root --protocol='tcp' -e "
+  DROP DATABASE IF EXISTS testdb1;
+  CREATE DATABASE testdb1;
+  USE testdb1;
+  CREATE TABLE products (
+      id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+      name VARCHAR(255) NOT NULL,
+      description VARCHAR(512)
+  );
+  ALTER TABLE products AUTO_INCREMENT = 101;
+  INSERT INTO products
+  VALUES (default,'scooter','Small 2-wheel scooter'),
+         (default,'car battery','12V car battery'),
+         (default,'12-pack drill','12-pack of drill bits with sizes ranging from #40 to #3'),
+         (default,'hammer','12oz carpenter s hammer'),
+         (default,'hammer','14oz carpenter s hammer'),
+         (default,'hammer','16oz carpenter s hammer'),
+         (default,'rocks','box of assorted rocks'),
+         (default,'jacket','water resistent black wind breaker'),
+         (default,'spare tire','24 inch spare tire');
+  CREATE TABLE orders (
+      order_id INTEGER NOT NULL AUTO_INCREMENT PRIMARY KEY,
+      order_date DATETIME NOT NULL,
+      customer_name VARCHAR(255) NOT NULL,
+      price DECIMAL(10, 5) NOT NULL,
+      product_id INTEGER NOT NULL,
+      order_status BOOLEAN NOT NULL
+  ) AUTO_INCREMENT = 10001;
+  INSERT INTO orders
+  VALUES (default, '2020-07-30 10:08:22', 'Jark', 50.50, 102, false),
+         (default, '2020-07-30 10:11:09', 'Sally', 15.00, 105, false),
+         (default, '2020-07-30 12:00:30', 'Edward', 25.25, 106, false);
+"
+
+statement ok
+create source mysql_source with (
+  connector = 'mysql-cdc',
+  hostname = '${MYSQL_HOST:localhost}',
+  port = '${MYSQL_TCP_PORT:8306}',
+  username = 'root',
+  password = '123456',
+  database.name = 'testdb1',
+  server.id = '5185'
+);
+
+statement ok
+create table products_1 ( id INT,
+ name STRING,
+ description STRING,
+ PRIMARY KEY (id)
+) from mysql_source table 'testdb1.products';
+
+statement ok
+create table orders_1 (*) from mysql_source table 'testdb1.orders';
+
+system ok
+psql -c "
```

Comment:
Can we separate tests for MySQL and Postgres into different files?

Reply:
Better to add some comments to explain the test scenario is "join three tables from 2 different databases" (or perhaps change table names e.g., `shipments1` -> `pg_ shipments`)

Reply:
BTW, why the table names are `<xx>1`..?